.github/ISSUE_TEMPLATE/issue-report.yml

```diff
@@ -0,0 +1,163 @@
+      placeholder: |
+        1. Sign in on the OpenBullet2 web client, with an account, using a browser of your choice.
+        2. Navigate to Wordlists.
+        3. Select "Add from local file or Upload (remote)".
+        4. Enter a name for the Wordlist and complete other fields.
+        5. Add the Wordlist.
+        6. The file is in a folder and accessible by editor but can't be accessed from OpenBullet2.
+    validations:
+      required: true
+  - type: textarea
+    id: actual-behavior
+    attributes:
+      label: What is the current _bug_ behavior?
+      description: Write down the incorrect behavior that currently happens after following the reproduction steps.
+      placeholder: |
+        The login disclaimer on the sign in screen has the server name appended to the text. The text shown is: "I am a cool disclaimer!".
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: What is the expected _correct_ behavior?
+      description: Write down the correct expected behavior that is supposed to happen after following the reproduction steps.
+      placeholder: |
+        The login disclaimer on the sign in screen should only show the configured text. The text that should be shown is: "I am a cool disclaimer!".
+    validations:
+      required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version of the software
+      description: What version of OpenBullet2 are you using?
+      options:
+        - 0.3.0
+        - 0.2.5
+        - Older*
+      default: 0
+    validations:
+      required: true
+  - type: textarea
+    id: environment-information
+    attributes:
+      label: Environment
+      description: |
+        Accurately fill in as much environment details as possible. If a certain environment field is not shown in the template below, but you consider useful information, please include it.
+        Examples:
+        - **OS**: [e.g. Debian 12, Windows 11]
+        - **Linux Kernel**: [e.g. none, 5.15, 6.1, etc.]
+        - **Virtualization**: [e.g. Docker, KVM, LXC]
+        - **Browser**: [e.g. Firefox 91, Chrome 93, Safari 13, native client]
+        - **Installed Plugins**: [...]
+        - **Reverse Proxy**: [e.g. none, nginx, apache, cloudflared, etc.]
+        - **Base URL**: [e.g. none, yes: /example]
+        - **Networking**: [e.g. Host, Bridge/NAT]
+        - **Storage**: [e.g. local, NFS, cloud]
+      value: |
+        - OS:
+        - Linux Kernel:
+        - Virtualization:
+        - Browser:
+        - Plugins:
+        - Reverse Proxy:
+        - Base URL:
+        - Networking:
+        - Storage:
+      render: markdown
+    validations:
+      required: true
+  - type: markdown
+    id: general-information-logs
+    attributes:
+      value: |
+        When providing logs, please keep the following things in mind:
```

Comment:
I would add information on where to get the logs. Currently, the web client log files can be found in the `UserData/Logs` folder in a format such as `log-20240613.txt`. The native client has a `crash.log` file in the main folder only when a crash happens.